hardhat.config.js

```diff
@@ -23,20 +23,24 @@ module.exports = {
   namedAccounts: {
     deployer: {
       goerli: 0,
+      sepolia: 0,
       mainnet: 0,
       mainnet_test: 0,
     },
     rewardsHolder: {
       goerli: "0xCe692F6fA86319Af43050fB7F09FDC43319F7612",
+      sepolia: "0xCe692F6fA86319Af43050fB7F09FDC43319F7612",
```

Comment:
I'm not sure if the value here is ok. I'm using the same value as in `goerli`, because in https://github.com/threshold-network/merkle-distribution/pull/44 the value for `goerli` was set to the same as for `ropsten`.

Reply:
That's ok!